psi4/src/psi4/libqt/qt.h

```diff
@@ -86,8 +86,10 @@ void mat_print(double **A, int rows, int cols, std::string out_fname);
 
 void timer_init(void);
 void timer_done(void);
-void timer_on(const char *key);
-void timer_off(const char *key);
+void timer_on(std::string key);
+void timer_off(std::string key);
```

Comment:
I don't think the timer is intended to be used for ultra fine grained purposes but, out of paranoia, shouldn't the key be passed by reference to avoid the tiny amount of copying overhead?  It's not likely to be an issue, but I see no harm in doing that, at first glance.